SPEC.md

```diff
@@ -266,9 +266,11 @@ public class Car
 }
 ```
 
+If the specified property name conflicts with a field name of the same class, the property will get precedence over the field during serialiation and deserialization.
```

Comment:
Can you address serialization and deserialization separately, since you can annotate properties that only have getters (as above), and similarly setters.